features/fixtures/policy_and_engagement_signup_content_item.json

```diff
@@ -0,0 +1,43 @@
+{
+  "base_path": "/policy-papers-and-consultations/email-signup",
+  "content_id": "5a4dc517-57cf-4dd6-873f-f1d29f6d540c",
+  "document_type": "finder_email_signup",
+  "title": "Policy papers and consultations",
+  "description": "You'll get an email each time policy papers and consultations are published.",
+  "details": {
+    "filter": {
+      "content_purpose_supergroup": "policy-papers-and-consultations"
+    },
+    "email_filter_by": null,
+    "email_filter_name": null,
+    "subscription_list_title_prefix": "Policy papers and consultations ",
```

Comment:
The trailing space is no longer required for `subscription_list_title_prefix` so lets remove it.